lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -67,17 +68,33 @@ def build_index(self, server_config, dates):
             self.index_version,
             self.index_prefix,
         )
-        index = "/"
+        index = ""
         for d in dates:
             index += f"{idx}{d},"
-        return index
+        if not index:
+            drb = Dataset.attach(controller="node", name="drb")
+            index_map = Metadata.getvalue(dataset=drb, key="server.index-map")
+            for key, values in index_map.items():
+                if self.payload["run_id"] in values:
+                    index = f"{key}"
+        return f"/{index}"
+
+    def build_index_from_metadata(self, find_metadata):
```

Comment:
I don't see a reference to `find_metadata` in this function; can it/should it be removed?

I see only one call to `build_index_from_metadata()`, and it's neglecting to specify a second parameter....

Reply:
Fixed